Mathlib/Algebra/Algebra/Operations.lean

```diff
@@ -62,49 +67,53 @@ end SubMulAction
 
 namespace Submodule
 
-variable {ι : Sort uι}
-variable {R : Type u} [CommSemiring R]
+section Module
 
-section Ring
-
-variable {A : Type v} [Semiring A] [Algebra R A]
-variable (S T : Set A) {M N P Q : Submodule R A} {m n : A}
+variable {R : Type u} [Semiring R] {A : Type v} [Semiring A] [Module R A]
 
-/-- `1 : Submodule R A` is the submodule R of A. -/
+/-- `1 : Submodule R A` is the submodule `R ∙ 1` of A. -/
```

Comment:
Okay, I would tentatively add this docstring. Would you like to open the issue, @Vierkantor?
```suggestion
/-- `1 : Submodule R A` is the submodule `R ∙ 1` of A.

TODO: potentially change this back to `LinearMap.range (Algebra.linearMap R A)`
once a version of `Algebra` without the `commutes'` field is introduced.

https://github.com/leanprover-community/mathlib4/pull/18092#issuecomment-2431443879
-/
```

Reply:
Ok! Created #18110. 

```suggestion
/-- `1 : Submodule R A` is the submodule `R ∙ 1` of A.
TODO: potentially change this back to `LinearMap.range (Algebra.linearMap R A)`
once a version of `Algebra` without the `commutes'` field is introduced.
See issue #18110.

https://github.com/leanprover-community/mathlib4/pull/18092#issuecomment-2431443879
-/
```

Reply:
Great! Since the issue refers to this PR, let me remove the link here.
```suggestion
/-- `1 : Submodule R A` is the submodule `R ∙ 1` of A.
TODO: potentially change this back to `LinearMap.range (Algebra.linearMap R A)`
once a version of `Algebra` without the `commutes'` field is introduced.
See issue #18110.
-/
```